main.tf

```diff
@@ -0,0 +1,175 @@
+terraform {
+  required_version = "~> 1.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+    template = {
+      source  = "hashicorp/template"
+      version = "~> 2.2.0"
+    }
+  }
+}
+
+
+provider "aws" {
+  region = var.aws_region
+  assume_role {
+    role_arn = var.aws_role_arn
+  }
+}
+
+locals {
+  prefix             = var.prefix
+  availability_zones = ["ap-southeast-1a", "ap-southeast-1b", "ap-southeast-1c"]
```

Comment:
Availability zones, should be `${var.aws_region}a" ? Or any reason why this cannot be?

Reply:
It seems is duplicated, right? maybe this constant can be reused?